Sources/Status.swift

```diff
@@ -67,72 +67,72 @@ public enum Status {
 extension Status {
     public init(statusCode: Int, reasonPhrase: String? = nil) {
         if let reasonPhrase = reasonPhrase {
-            self = other(statusCode: statusCode, reasonPhrase: reasonPhrase)
+            self = .other(statusCode: statusCode, reasonPhrase: reasonPhrase)
         } else {
             switch statusCode {
-            case `continue`.statusCode:                    self = `continue`
-            case switchingProtocols.statusCode:            self = switchingProtocols
-            case processing.statusCode:                    self = processing
+            case Status.`continue`.statusCode:                    self = .`continue`
```

Comment:
Didn't work without it for me, since `statusCode` is just an `Int`, there's no type inference to help us AFAIK.